### ISSUE TITLE
Formatting errors

### ISSUE THREAD

**Issue by floffimedia** [pluginkollektiv/crazy-lazy#16]
Thanks so much for your work on this wonderful plugin. It's definitely a must-have for my site.

I just noticed that there seem to be some minor formatting errors in version 1.0.0. The "style" attribute should come before the closing slash of the img element. And could you please have a look at the duplicate / missing spaces as well?

This is a real life example from my site running WP 4.6.

``` html
<img   class="crazy_lazy wp-image-3686 size-full "src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="  data-src="https://floffi.media/images/Mr-Robot-Elliot-01.jpg" alt="Mr Robot Elliot 01" width="1500" height="844" srcset="https://floffi.media/images/Mr-Robot-Elliot-01.jpg 1500w, https://floffi.media/images/Mr-Robot-Elliot-01-855x481.jpg 855w" sizes="(max-width: 1500px) 100vw, 1500px" / style="display:none">
```

**Comment by 2ndkauboy** [comment]
Thanks for your bug report @floffimedia!

I think I found the issue with the trailing slash. I'll try to fix that later and make an update to the plugin repo.

Regarding the duplicate/missing space: Duplicates shouldn't be an issue. But missing space would be an issue. I just can't see them in your example. And after a second look I also see the missing space before the src attribute.

**Comment by 2ndkauboy** [comment]
I have quickly fixed the issue but haven't triggered a new release. It would be great if you can just test it.

**Comment by floffimedia** [comment]
@2ndkauboy Thank you very much for the quick response. I'm sorry I couldn't get back to you earlier; I was in Cuba and it's really hard to get internet access in that country.

Version 1.0.3 works great with thumbnails:

```html
<img  width="600" height="400"  class="crazy_lazy attachment-medium size-medium wp-post-image" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="   data-src="https://floffi.media/images/Mr-Robot-Elliot-01-600x400.jpg"  alt="Mr Robot Elliot 01"  style="display:none" />
```

Unfortunately, it doesn't work with regular post images inside the_content anymore. This is the image from my initial comment:

```html
<img class="wp-image-3686 size-full" src="https://floffi.media/images/Mr-Robot-Elliot-01.jpg" alt="Mr Robot Elliot 01" width="1500" height="844" srcset="https://floffi.media/images/Mr-Robot-Elliot-01.jpg 1500w, https://floffi.media/images/Mr-Robot-Elliot-01-855x481.jpg 855w" sizes="(max-width: 709px) 85vw, (max-width: 909px) 67vw, (max-width: 1362px) 62vw, 840px" />
```

Maybe that's just me, but I've tested it with the Twenty Sixteen theme activated and all other plugins disabled.

**Comment by 2ndkauboy** [comment]
@floffimedia: Thanks for your bug report and the HTML code. I think I found a solution for this one.

It would really help, if you could just test the current version here on Github. If this works for you and I find no issues with other test cases, I can release a 1.0.4 version.

**Comment by floffimedia** [comment]
@2ndkauboy Thank you, that brings the functionality back. The markup is still not very pretty though:

´´´html
<img   class="crazy_lazy wp-image-3686 size-full" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="   data-src="https://floffi.media/images/Mr-Robot-Elliot-01.jpg"  alt="Mr Robot Elliot 01" width="1500" height="844" srcset="https://floffi.media/images/Mr-Robot-Elliot-01.jpg 1500w, https://floffi.media/images/Mr-Robot-Elliot-01-855x481.jpg 855w" sizes="(max-width: 1500px) 100vw, 1500px" / style="display:none" >
´´´

**Comment by 2ndkauboy** [comment]
@floffimedia thanks for testing!

With the "unpretty" markup, you mean the few additional whitespaces? They are really no issue for any browser. And removing them would not only waist precessing time, but might also introduce new issues.

**Comment by 2ndkauboy** [comment]
Ah I just saw your issue. That is again the initial issue. It was not easy to fix, but I hope I found a solution now. Could you pleas test the current version here from Github again?

**Comment by floffimedia** [comment]
@2ndkauboy It works perfectly. Thank you so much! I'm sure it must have been complicated, it's definitively too complicated for me :)

This is the output I get:

```html
<img   style="display:none"  class="crazy_lazy wp-image-3686 size-full" src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="   data-src="https://floffi.media/images/Mr-Robot-Elliot-01.jpg"  alt="Mr Robot Elliot 01" width="1500" height="844" srcset="https://floffi.media/images/Mr-Robot-Elliot-01.jpg 1500w, https://floffi.media/images/Mr-Robot-Elliot-01-855x481.jpg 855w" sizes="(max-width: 1500px) 100vw, 1500px" />
```

**Comment by Zodiac1978** [comment]
Could this be closed?

**Comment by 2ndkauboy** [comment]
Closed in [v1.0.4](https://github.com/pluginkollektiv/crazy-lazy/releases/tag/v1.0.4)
